Origin:                  https://intestinate.com/pilfs

--- gcc-9.1.0.orig/gcc/config.gcc	2019-07-15 22:51:55.794365183 -0400
+++ gcc-9.1.0/gcc/config.gcc	2019-07-15 22:59:22.502945967 -0400
@@ -1189,6 +1189,13 @@
 	    tmake_file="$tmake_file arm/t-linux-androideabi"
 	    ;;
 	esac
+	case ${target} in
+	arm*-*-*eabihf)
+	    with_cpu=${with_cpu:-cortex-a53}
+	    with_fpu=${with_fpu:-neon-fp-armv8}
+	    with_float=${with_float:-hard}
+	    ;;
+	esac
 	# The EABI requires the use of __cxa_atexit.
 	default_use_cxa_atexit=yes
 	with_tls=${with_tls:-gnu}
